README.rst

```diff
@@ -265,6 +265,11 @@ A record is a dict with the "permissions" and "data" keys.
     # Or retrieve all records.
     records = client.get_records(collection='todos', bucket='default')
 
+   # Or retrive records page by page.
+   for page in client.get_paginated_records(collection='todos', bucket='dafult'):
```

Comment:
default

Reply:
Done! Sorry for the typos, fast typing yields mistakes from time to time hahaha